docs/How-To-Integrate.md

```diff
@@ -46,9 +46,11 @@ In our example Rails application, we have used [Omniauth](https://github.com/int
 Once you have obtained an access token for the end user, you need to set up your web app as a notifications source for the specified flow. This is described in the next section of this guide.
 
 <div id="/create-integration"></div>
-## 3. Add your app as an inbox source
+## 3. Add your app as a source
 
-Before you can post messages to a flow, you need to use the access token (obtained in the previous step) and the flow parameters (that your setup endpoint received as query parameters) to add your app to the flow's team inbox as a [source](sources). Once done, you will obtain a flow token that is used to post notifications ([messages](messages)) into the flow's team inbox. When a source is added to a flow, it will also show up as an inbox filter in that flow. The source will also identify the external entity (eg. a repo for GitHub or a project for a project management tool) that your OAuth application has tied to the particular flow.
+Before you can post messages to a flow, you need to use the access token (obtained in the previous step) and the flow parameters (that your setup endpoint received as query parameters) to add your app to the flow's team inbox as a [source](sources). Once done, you will obtain a `flow_token` that is used to post notifications ([messages](messages)) into the flow's team inbox.
```

Comment:
The wording implies that flow token wouldn't have any otheruses than posting notifications(?) to team inbox.